src/Models/HydrostaticFreeSurfaceModels/compute_hydrostatic_free_surface_tendencies.jl

```diff
@@ -122,7 +122,7 @@ function compute_hydrostatic_free_surface_tendency_contributions!(model, kernel_
                     c_tendency,
                     grid,
                     active_cells_map,
-                    args;
+                    args...;
```

Comment:
This went in previously to fix an O(100)x slow down on CPU, so we should test whether the slow down is present in this PR